app/docs-v3/auth-api.html

```diff
@@ -197,6 +204,44 @@ <h4>Examples</h4>
 		</div>
 	</div>
 
+	<!-- example 6 -->
+
+	<div class="accordion-group">
+		<div class="accordion-heading">
+			<a class="accordion-toggle collapsed" data-toggle="collapse" data-parent="#example5" href="#example5">
+				Example: Catalog and Missing Catalog
+			</a>
+		</div>
+		<div id="example5" class="accordion-body collapse">
+			<div class="accordion-inner">
+				<div class="bs-docs-example bs-docs-example-tag">
+					<pre>GET https://api.mindflash.com/api/v3/auth?id=9999&catalogs=123,789</pre>
+				</div>
+				<pre class="prettyprint">
+Response: 200
+{
+  userId: 9999,
+  userDisplayName: "Joe Black",
+  userEmail: "joe.black@yourorganization.com",
+  userFirstName: "Joe",
+  userLastName: "Black",
+  dashboardUrl: "http://yourorg.mindflash.com/CreateCookie.aspx?sessionID2=AcAVGBh34343SD"
+  catalogs: [
+    {
+      id: 123,
+      name: "How to Hang a Door",
+      url: "https://yourorg.mindflash.com/CreateCookie.aspx?sessionID2=B4dawfjkCVCFct"
```

Comment:
ditto

Reply:
@zadevictor - take a look :) make sure I got them this time :)